src/app/archive/archive_lib/test.ml

```diff
@@ -191,9 +191,11 @@ let%test_module "Archive node unit tests" =
                           Processor.Block.load conn ~id
                         in
                         if
-                          Transition_frontier.Breadcrumb.blockchain_length
-                            breadcrumb
-                          > Unsigned.UInt32.of_int 1
+                          Unsigned.UInt32.compare
+                            (Transition_frontier.Breadcrumb.blockchain_length
+                               breadcrumb)
+                            (Unsigned.UInt32.of_int 1)
+                          > 0
```

Comment:
I think this would be clearer:

```ocaml
let blockchain_length = Transition_frontier.Breadcrumb.blockchain_length breadcrumb in
Unsigned.UInt32.Infix(blockchain_length > Unsigned.UInt32.one
```

Reply:
`Unsigned.UInt32.Infix.( < )` doesn't exist, unfortunately :(